conans/client/conf/__init__.py

```diff
@@ -338,6 +338,19 @@ def request_timeout(self):
         except ConanException:
             return None
 
+    @property
+    def revisions_enabled(self):
+        try:
+            revisions_enabled = get_env("CONAN_CLIENT_REVISIONS_ENABLED")
```

Comment:
Because that exactly what I'm trying to avoid. If I set the env var and then I forget about the config entry, several `TestClients` running at the same time will mix the activation of revisions. This is a general issue of that env var management. But in this case is harder.

Reply:
ok, so you could enable the configuration of each client instead of using env vars. That way each client will work with or without revisions independently and we will keep the same code as any other config entry